apps/cowswap-frontend/src/modules/trade/pure/TradeConfirmation/index.tsx

```diff
@@ -38,8 +38,11 @@ export interface TradeConfirmationProps {
   recipient?: string | null
   buttonText?: ReactNode
   children?: (restContent: ReactElement) => ReactElement
+  'data-click-event'?: string
 }
 
+// TODO: Break down this large function into smaller functions
+// eslint-disable-next-line max-lines-per-function
```

Comment:
It's better to address the issue in this PR.
https://biratkirat.medium.com/step-8-the-boy-scout-rule-robert-c-martin-uncle-bob-9ac839778385